ts/python/project.test.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, it } from "https://deno.land/std@0.206.0/testing/bdd.ts";
+import * as garn from "./mod.ts";
+import { getStorePath, nixpkgsInput } from "../testUtils.ts";
+import { assert } from "https://deno.land/std@0.206.0/assert/mod.ts";
+import { renderFlakeFile } from "../nix.ts";
+import { formatFlake } from "../internal/runner.ts";
+import * as path from "https://deno.land/std/fs/copy.ts";
+import { existsSync } from "https://deno.land/std@0.201.0/fs/exists.ts";
+
+describe("mkPythonProject", () => {
+  it("produces correct store path structure", () => {
```

Comment:
I think it'd be nice to frame the tests more from the perspective of a `garn` user, e.g.:

```suggestion
  it("allows building a python project and running scripts from it", () => {
```

And then execute the script in the test and assert on its output. (`garn` users probably don't care that much about what ends up in the nix store, but they do care about being able to execute their python scripts.)